Tasks/AzurePowerShellV3/task.json

```diff
@@ -16,14 +16,13 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 3,
-        "Minor": 0,
-        "Patch": 5
+        "Minor": 1,
```

Comment:
Are there any Build or Release templates that uses this task and now needs to point to latest version?